test/unit/services/oooRequest.test.ts

```
@@ -0,0 +1,161 @@
+        it("should create OOO request", async function() {
+            const response = await createOOORequest(
+                validOooStatusRequests,
+                "test-username-1",
+                testUserId
+            );
+            expect(response).to.exist;
+            expect(response).to.deep.include({
+                type: validOooStatusRequests.type,
+                from: validOooStatusRequests.from,
+                until: validOooStatusRequests.until,
+                reason: validOooStatusRequests.reason,
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)